packages/amplify-category-api/src/provider-utils/awscloudformation/cfn-api-artifact-handler.ts

```diff
@@ -75,6 +79,59 @@ class CfnApiArtifactHandler implements ApiArtifactHandler {
     return serviceConfig.apiName;
   };
 
+  updateArtifacts = async (request: UpdateApiRequest): Promise<void> => {
```

Comment:
Shouldn't this function have a switch function to choose between APIGW and AppSync for the future?

Reply:
I thought about that but I was going to take the approach of not doing that until it's needed rather than a switch with only one case. But I can do that if you think we should go ahead and do it that way

Reply:
Okay, just add a TODO comment out there.

Reply:
added a TODO